appserver/data/data-core/src/main/java/fish/payara/jakarta/data/core/cdi/extension/JakartaDataExtension.java

```diff
@@ -79,7 +88,7 @@ public Set<Class<?>> getRepositoryClasses() {
         Set<Class<?>> repositories = new HashSet<>();
         //here is the place to start to use the classgraph api to get information from the classes that are 
         //annotated by the jakarta data annotation 
-        try (ScanResult result = new ClassGraph().enableAllInfo().scan()) {
+        try (ScanResult result = new ClassGraph().enableClassInfo().enableAnnotationInfo().scan()) {
```

Comment:
Do we need to include `.enableClassInfo()`?

If the goal is to just get specific annotated classes just `.enableAnnotationInfo()` should be enough?

Reply:
yes you are right after removing the enableClassInfo the implementation still worked well, so we don't nee the ClassInfo just the AnnotationInfo, good catch

Reply:
Already done!!